Source/Krypton Components/Krypton.Toolkit/Utilities/RoundedRectangleF.cs

```diff
@@ -18,21 +18,28 @@ namespace Krypton.Toolkit
     public class RoundedRectangleF
     {
         #region Variables
-        private readonly float _x;
-        private readonly float _y;
-        private readonly float _width;
-        private readonly float _height;
+        private readonly float _x, _y, _width, _height;
```

Comment:
This is a bad practice patter.. Please undo.

Reply:
Done